.idea/encodings.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should be commited to the repo, please remove whole .idea catalogue and add it to the .gitignore.